tests/Microsoft.Bot.Builder.Azure.Tests/CosmosDbPartitionStorageTests.cs

```diff
@@ -291,95 +294,250 @@ public async Task WritingNoStoreItemsDoesntThrow()
         [TestMethod]
         public async Task WaterfallCosmos()
```

Comment:
Can we change this name to something that describes what is being tested?

Reply:
Changed to `CosmosStorageCanHandleBotStateInWaterfallDialog`. Open to other ideas